keps/sig-node/3386-kubelet-evented-pleg/README.md

```diff
@@ -134,30 +134,31 @@ This might be a good place to talk about core concepts and how they relate.
   - Users can disable this feature to make kubelet use existing relisting based PLEG.
 - Another risk is the CRI implementation could have a buggy event emitting system, and miss pod lifecycle events.
   - A mitigation is a `kube_pod_missed_events` metric, which the Kubelet could report when a lifecycle event is registered that wasn't triggered by an event, but rather by changes of state between lists.
+  - While using the Evented implementation, the periodic relisting functionality would still be used with an increased interval which should work as a fallback mechanism for missed events in case of any disruptions.
```

Comment:
Please see if we would want to keep this line here about Generic PLEG relisting functionality as a fallback for missed events from CRI. (or rephrase it?)